webpack.config.js

```diff
@@ -159,4 +160,4 @@ module.exports = function(env) {
         buildConfig
       );
   }
-};
+} : {};
```

Comment:
Why are any changes required in this file? If anything I would recommend pushing these to a separate PR that describes why they're needed. I have a feeling this may have to do with the issue you created #1414, so we could discuss there as well.